mods/ctf_pvp_engine/ctf_chat/init.lua

```diff
@@ -148,6 +158,68 @@ minetest.register_chatcommand("team", {
 	end
 })
 
+minetest.register_chatcommand("access", {
+	description = "Open the team console, or run team command (see /team help)",
```

Comment:
The description seems to have been copy-pasted from the `/team` chat-command def. You might want to provide an actual description here :P

Reply:
It was.